installation/template/index.php

```diff
@@ -53,45 +53,37 @@
 	<head>
 		<jdoc:include type="metas" />
 		<jdoc:include type="styles" />
+		<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
```

Comment:
Do we really need shrink to fit? From memory this was just to fix a bug in iOS 9.2 and nothing else

Reply:
No, a quick web search says the same. We can remove shrink-to-fit=no.

Reply:
Thanks.

Reply:
Thank you